apps/fabric-website-resources/src/components/pages/ColorsPage.tsx

```diff
@@ -379,26 +383,47 @@ export class ThemeGeneratorPage extends BaseComponent<{}, IThemeGeneratorPageSta
       }
     }
 
+    const themeAsCode = ThemeGenerator.getThemeAsCode(abridgedTheme);
+
     return (
       <div>
         <h2 id="Output">Output</h2>
         <div className={'ms-themer-output-root'}>
-          <div>
-            <h3>JSON</h3>
-            <textarea
-              readOnly={true}
-              spellCheck={false}
-              value={JSON.stringify(ThemeGenerator.getThemeAsJson(abridgedTheme), void 0, 2)}
-            />
-          </div>
-          <div>
-            <h3>SASS</h3>
-            <textarea readOnly={true} spellCheck={false} value={ThemeGenerator.getThemeAsSass(abridgedTheme)} />
-          </div>
-          <div>
-            <h3>PowerShell</h3>
-            <textarea readOnly={true} spellCheck={false} value={ThemeGenerator.getThemeForPowerShell(abridgedTheme)} />
-          </div>
+          <Pivot styles={{ root: { padding: 10 } }} style={{ width: '100%' }}>
+            <PivotItem linkText="Code">
+              <p>
+                This code block initializes the theme you have configured above and loads it using the loadTheme utility function. Calling
+                loadTheme will automatically apply the configured theming to any Fabric controls used within the same app.
+              </p>
+              <textarea
+                readOnly={true}
+                spellCheck={false}
+                value={codeHeader + themeAsCode}
+                // value={codepenSamples}
+                style={{ width: '100%' }}
```

Comment:
IMO it looks nicer to have width:300px so it doesn't have all the extra whitespace inside the textbox. (If keeping the Sass section, it needs to be 500px instead.)